src/main/scala/com/typesafe/sbt/git/ConsoleGitRunner.scala

```diff
@@ -14,12 +14,15 @@ object ConsoleGitRunner extends GitRunner {
 	}
   private lazy val cmd = if(isWindowsShell) Seq("cmd", "/c", "git") else Seq("git")
 
+  // in order to enable colors we trick git into thinking we're a pager, because it already knows we're not a tty
+  val colorSupport = ("GIT_PAGER_IN_USE", "1")
```

Comment:
There is a flag we can grab to check if ANSI colors are enabled for this sbt.   We should check this value before turning this on, I think.   Let me check.

otherwise, looks good!